lms/djangoapps/instructor/views/api.py

```diff
@@ -867,11 +867,13 @@ def save_query(request, course_id):
         for query in existing_queries
         if (query != "working" and query != "")
     ]
-    success = data_access.save_query(course_id, clean_existing)
+    group = data_access.save_query(course_id, clean_existing)
+    group_title = group.title if group.title else u'Query saved at ' + group.created.strftime("%m-%d-%y %H:%M")
```

Comment:
I think it's preferable if the title isn't set in this way (if we're loading it in the GUI, we don't have a way of checking whether it's a user set title or whether it's a "Query Saved at ...." title). This is important because if it's not a user set title, we would want to display the query components rather than "Query saved at...". Is it possible to toggle the display in the front-end?